lightning/src/util/config.rs

```diff
@@ -245,6 +265,7 @@ impl Default for UserConfig {
 			own_channel_config: ChannelHandshakeConfig::default(),
 			peer_channel_config_limits: ChannelHandshakeLimits::default(),
 			channel_options: ChannelConfig::default(),
+			accept_forwards_to_priv_channels: false,
```

Comment:
Would this be better suited for the ChannelConfig on channel creation, to make it configurable per-channel?

Reply:
Hmmm, is there a specific use-case you have in mind? I feel like this is kinda just a "do you want forwarding" knob, the only reason its not exactly that is we should generally never allow the disabling of forwarding out over public channels, cause its realllllyyyy anti-social.

Reply:
No particular use-case in mind. Seems a little more "future-proof" but eh. Resolving